### ISSUE TITLE
Modifying binary classification to Multi-Class Classification (Logistic Regression)

### ISSUE THREAD

**Issue by ahmed2180** [tensorflow/tfx#2503]
I working on a machine learning classification problem for logistic regression, initially there is a code for LG for binary classification with 2 classes.

[https://github.com/datafleets/horizontal-federated-learning-blog/blob/master/horizontal-fl.ipynb](url)

I want to change this code to work with Multi-Class Classification with six (classes), for example with six activities (walking, walkingupstairs, walkingdownstairs, sitting, standing, lying)
[https://www.kaggle.com/uciml/human-activity-recognition-with-smartphones](url)
 instead of binary classification.
**Please let me know how to make it work for a multi-class classification?**

**Comment by gowthamkpr** [comment]
@ahmed2180 This is not a tfx issue. I would recommend you to post this on stackoverflow. 
You can take a look at this [repo](https://github.com/vijayg15/Keras-MultiClass-Image-Classification) where multiclass classificaton has been implemented.

The only difference between binary and multi-class classification is to configure the outputs of the output layer(softmax layer) and then train the model.

**Comment by gowthamkpr** [comment]
Closing this issue as it has been awaiting response for more than 2 weeks. Please add additional comments for us to open this issue again. Thanks!

**Comment by albertnanda** [comment]
@gowthamkpr How do you process multlabel data in tft transform. I am trying to convert it to one hot encoding:
tft.compute_and_apply_vocabulary(inputs[LABEL_KEY]), this part works. How can i convert it to one-hot vectors?
